### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by giporro**

### CODE REVIEW

File: openapi/openapi_V1.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+basePath: /api/v1/payment
+schemes:
+  - https
+
+tags:
+  - name: API  Payment
+    description: API  Payment
+
+paths:
+  /check/{noticeNumber}:
```
Comment (fabriziopapi) [inline openapi/openapi_V1.yaml:16]: How is possible to use the noticeNumber as key? The notice number is not unique.
I would expect to have the message id.

File: openapi/openapi_V1.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          type: string      
+      responses:
+        200:
+          description: check if the message has been paid
+          schema:
+            $ref: '#/definitions/ResponseCheck'
+definitions:
+  ResponseCheck:
+    type: object
+    additionalProperties:
```
Comment (fabriziopapi) [inline openapi/openapi_V1.yaml:36]: Why we are using additionalProperties instead of definign the message reponse?

File: openapi/openapi_V1.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      summary: Check by notice number
+      operationId: checkAssistenza
+      parameters:
+        - name: noticeNumber
+          in: path
+          required: true
+          description: payment's notice number
+          type: string      
+      responses:
+        200:
```
Comment (fabriziopapi) [inline openapi/openapi_V1.yaml:29]: This endpoint return always 200?

File: openapi/openapi_V1.yaml (hunk 2 above)
Comment (gparente3) [reply openapi/openapi_V1.yaml]: This bug was fixed in the latest commit

**Pull request merged**